src/html/partials/key-factors.html

```diff
@@ -1,4 +1,37 @@
 <section class="factors-section">
+  
   <h2 class="sm-title">Why People Don't Adhere</h2>
   <p class="bg-paragraph">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
+
+  <div class="factors-section__human-figure-wrapper">
+    <div class="factors-section__human-figure-gear"></div>
+    <div class="factors-section__human-figure-heart"></div>
+    <div class="factors-section__human-figure-dollar"></div>
+    <div class="factors-section__human-figure-tally"></div>
+  </div>
+
+  <div class="factors-section__msg">
+    <div class="factors-section__gear"></div>
+    <h3>Comprehension</h3>
+    <p>“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly questions.”</p>
+  </div>
+
+  <div class="factors-section__msg">
+    <div class="factors-section__heart"></div>
+    <h3>Denial</h3>
+    <p>“I feel just fine. I don’t know why that doctor thinks I’m sick. Why would I take medicine that I don’t need?”</p>
+  </div>
+
+  <div class="factors-section__msg">
+    <div class="factors-section__dollar"></div>
+    <h3>Cost</h3>
+    <p>“I took my prescription to the pharmacy but they said it would cost several hundred dollars! I just couldn’t afford it.”</p>
+  </div>  
+
+  <div class="factors-section__msg">
+    <div class="factors-section__tally"></div>
+    <h3>Complexity</h3>
+    <p>“Taking 15 medications every day is tough. Especially since they all have different instructions. How am I supposed to keep track of everything?”</p>
+  </div>
```

Comment:
Another way to look at these pieces of messaging is as a "list" of factors (since this is the key factors section). So with that in mind, I'd recommend using a `ul` for this, e.g.

```
<ul class="factors-section__list">
  <li class="factors-section__list__item">
    div/h3/p
  </li>
  ...
</ul>
```

Reply:
Done.